components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityUtil.java

```diff
@@ -1962,4 +1962,20 @@ public static boolean isSCIM2UserMaxItemsPerPageEnabled() {
         }
         return Boolean.parseBoolean(scim2UserMaxItemsPerPageEnabledProperty);
     }
+
+    /**
+     * Read the SCIM User Endpoint Consider Server Wide config and returns it.
+     *
+     * @return If SCIM User Endpoint Consider Server Wise Config is enabled.
+     */
+    public static boolean isConsiderServerWideUserEndpointMaxLimitEnabled() {
```

Comment:
We can have the method in here `https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/blob/master/components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/SCIMCommonUtils.java` 

We can move to the framework identityUtil if it is identity related concern.

Reply:
will do

Reply:
or else any commonly used methods are moved to here. As this methods looks useful in scim user manager, we can have the method implemented in that level.